build/NuSpecs/readme.txt

```diff
@@ -8,15 +8,23 @@ Don't forget to set XamlControlsResources as your Application resources in App.x
         </Application.Resources>
     </Application>
 
-or if you have other resources then add XamlControlsResources at the top as a merged dictionary:
+If you have other resources then we recommend you add those to the XamlControlsResources' MergedDictionaries.
+This works with the platform's resource system to allow overrides of the XamlControlsResources resources.
+
+<ResourceDictionary
```

Comment:
Not sure if ResourceDictionary is the correct "header" for this sample if the sample below is inside Application.Resources. Wouldn't `Application` make more sense here?

Reply:
This is a little confusing. What is the freestanding Resource Dictionary showing ?

Reply:
I think this would be the "container" of the snippet below except XAML below is inside an application tag, not a resource dictionary.